ui/packages/ui/src/main/webapp/js/application.js

```diff
@@ -102,8 +102,18 @@ define([
         'click button': 'logout',
       },
       logout: function() {
-        window.location =
-          '../logout/?prevurl=' + encodeURI(window.location.pathname)
+        $.get('../services/logout/actions', function(data) {
+          var actions = JSON.parse(data)
+          var doLogout = function(action) {
+            window.location.href = action.url
+          }
+
+          if (actions.length !== 0) {
+            actions.forEach(function(action) {
+              doLogout(action)
+            })
```

Comment:
❓ If we have multiple actions, wont this just set the `window.location` to the last action's url?

Reply:
This is doing what we used to do when we had no-prompt passed in https://github.com/codice/ddf/blob/master/ui/packages/security-servlet-logout/src/main/webapp/js/main.js#L30-L31